elements/designer-element-highlight/designer-element-highlight.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+
+<html>
+  <head>
+    <link rel="import" href="../../../polymer/polymer.html">
+  </head>
+  <body>
+
+    <dom-module id="designer-element-highlight">
+      <style>
+        :host {
+          display: block;
+          position: absolute;
+          box-sizing: border-box;
+        }
+        :host.before {
```

Comment:
Just an observation, but reading these class names immediately evoked the `::before` and `::after` psuedo-elements for me, but after thinking about it for a second it seemed that the meaning of the names was not related to those psuedo-elements.